apps/console/src/features/connections/components/edit/settings/authenticator-settings.tsx

```diff
@@ -115,7 +115,7 @@ const AUTHORIZED_REDIRECT_URLS: string[] = [ "callbackUrl", "callBackUrl" ];
  */
 const commonAuthenticators: string[] = [
     ConnectionManagementConstants.GOOGLE_OIDC_AUTHENTICATOR_ID,
-    ConnectionManagementConstants.MICROSOFT_AUTHENTICATOR_ID,
+    ConnectionManagementConstants.MS_LIVE_AUTHENTICATOR_ID,
```

Comment:
Is this an intended change?

Reply:
Yes. This is to hide the Microsoft Authenticator.